### ISSUE TITLE
Suggestions/feedback on collaboration

### ISSUE THREAD

**Issue by sbose78** [redhat-developer/service-binding-operator#405]
As we embark on https://github.com/application-stacks/service-binding-specification/issues/16 , 
In your opinion, what would be the best way to collaborate so that we can track bugs, enhancements and milestones:
- What has worked well so far ?
- What can we do better ( and how if you have suggestions ) ?
- What should we start doing? ( weekly design meetings, short stand-ups, slack? )

The goal of starting this thread is to understand what we should be doing as a team so that we can be successful in 'developing' Service Binding as well as having it consumed in our respective organizations/projects  ( cc @metacosm @pdettori @navidsh  @arthurdm and @ZhuangYuZY , would love to have your feedback )

**Comment by ZhuangYuZY** [comment]
Yes, it could be great to have kinds of communication with you as a team, then we could have same understanding on roadmap, milestones, features, critical bugs and consumption. For me, I am open for any kinds of communication. I will vote for weekly meeting and slack from my side. Thank you.

**Comment by arthurdm** [comment]
hey @sbose78 - first, thanks for all the great work and frictionless collaboration from you and the team.  👍 

I think the GitHub issues in this repo and the spec repo have really been working well in terms of tracking work and knowing what's pending.  

+1 for design meetings to discuss these.  I guess some topics we discuss at the service binding specification calls, but would be good to also have a dedicated SBO-focused call.  

Slack would be great for quick questions, clarifications, etc.  There's the private channel I created on `ibm-cloud` slack (I added you, but not sure if you were able to join) - I guess you were thinking something more public?
